cyences_app_for_splunk/default/macros.conf

```diff
@@ -113,6 +113,77 @@ iseval = 0
+| eval tmp_qualys_status = qualys_status, qualys=case(tmp_qualys_status == "Red", 3,tmp_qualys_status=="Orange",2, tmp_qualys_status=="Yellow",1, tmp_qualys_status=="Green",0) \
+| fieldformat qualys=tmp_qualys_status \
+| eval tenable_status=case(isnull(tenable_last_event), "-", critical_active_vuln>0, "Red", high_active_vuln>0, "Orange", medium_active_vuln>0, "Yellow", 1==1, "Green") \
+| eval tmp_tenable_status = tenable_status, tenable=case(tmp_tenable_status == "Red", 3,tmp_tenable_status=="Orange",2, tmp_tenable_status=="Yellow",1, tmp_tenable_status=="Green",0) \
+| fieldformat tenable=tmp_tenable_status \
+| eval defender_status=case(isnull(defender_last_event), "-", 1==1, RTP_state) \
+| eval tmp_defender_status = defender_status, defender=case(tmp_defender_status == "Enabled", 0,tmp_defender_status=="Disabled",1,true(),tmp_defender_status) \
+| fieldformat defender=tmp_defender_status \
+| eval sophos_status=case(isnull(sophos_last_event), "-", isnull(sophos_status), "RTP Enabled", 1==1, sophos_status) \
+| eval tmp_sophos_status = sophos_status, sophos=case(tmp_sophos_status == "RTP Enabled", 0,tmp_sophos_status=="RTP Disabled",1,tmp_sophos_status=="Service not Running",2) \
+| fieldformat sophos=tmp_sophos_status \
+| eval crowdstrike_status=if(isnull(crowdstrike_last_event), "-", "Present") \
+| eval tmp_crowdstrike_status = crowdstrike_status, crowdstrike=case(tmp_crowdstrike_status == "Present", 0) \
+| fieldformat crowdstrike=tmp_crowdstrike_status \
+| eval ip_drilldown=mvjoin(ip, ","), hostname_drilldown=mvjoin(hostname, ","), user_drilldown=mvjoin(user, ",") \
+| eval active_vul=coalesce(active_vuln, ACTIVE), total_vul=coalesce(total_vuln, TOTAL_VULNS) \
+| table _time, uuid, ip, hostname, mac_address, user, os, lansweeper, qualys, tenable, active_vul, total_vul, sophos, defender, crowdstrike, tmp_*, *_last_event, ip_drilldown, hostname_drilldown, user_drilldown \
+| fields - crowdstrike_status, sophos_status, defender_status, tenable_status, qualys_status, lansweeper_status
+iseval = 0
+
+[cs_lansweeper_inventory_fill_search]
+definition = eval lansweeper_id=coalesce(AssetID,id) | dedup lansweeper_id \
+| eval fqdn=coalesce('assetBasicInfo.fqdn',FQDN) | eval hostname=lower(mvjoin(mvdedup(mvappend(AssetName, fqdn)),"~~")), ip=lower(IPAddress), mac_address=lower(Mac) \
+| rename host as lansweeper_collected_by, site_name as Site, AssetTypename as AssetType, Statename as lansweeper_state, Userdomain as Domain, AssetGroup as GroupName, OScode as OSVersion, Username as lansweeper_user, BuildNumber as BuildNumber, version as AssetVersion, OS as lansweeper_os, fqdn as lansweeper_fqdn \
+| table _time, lansweeper_collected_by, hostname, ip, mac_address, lansweeper_id, Site, AssetType, lansweeper_state, Domain, GroupName, OSVersion, BuildNumber, AssetVersion, lansweeper_user, lansweeper_os, Description, IPLocation, lansweeper_fqdn \
+| append [| inputlookup cs_lansweeper_inventory | rename time as _time] \
```

Comment:
We don't need to append old entries in Lansweeper as we are always collecting all the entries.